### PR TITLE
Puppet

### PR DESCRIPTION
**Pull request by xaiki**
rewrite parsing bac.gob with puppet.

### CODE REVIEW

File: puppet/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:alpine as build
+    MAINTAINER Niv Sardi <x@filtra.me>
```
Comment (seppo0010) [inline puppet/Dockerfile]: is there any reason for the indentation? I don't think I've seen it in Dockerfiles 🤔

File: puppet/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+COPY package.json tsconfig.json run.sh ./
+COPY src ./src
+RUN yarn
```
Comment (seppo0010) [inline puppet/Dockerfile]: I think you can run yarn before copying src to have node_modules cache when you change src

**Review by seppo0010 — CHANGES_REQUESTED** [review]
I would like you to use a different repo as it looks like a different project than what currently exists

File: puppet/Dockerfile (hunk 1 above)
Comment (xaiki) [reply puppet/Dockerfile]: emacs thought it was sexeh, fixed.

File: puppet/Dockerfile (hunk 2 above)
Comment (xaiki) [reply puppet/Dockerfile]: fixd in 79a93b6

**Comment by xaiki** [comment]
about the different repo:
the plan is to merge and compare the data from the csv/api/datos abiertos to the scrapped one, it doesn't make sense to me to move it out as we'll need to merge it back in soon enough. we even have an issue for this here https://github.com/odia/buenosairescompras/issues/3


rationale: it's the same data, it should be in the same repo…

File: puppet/Dockerfile (hunk 1 above)
Comment (xaiki) [reply puppet/Dockerfile]: ab7ee13

**Comment by xaiki** [comment]
Fixes #3

**Pull request merged**